src/methods.jl

```diff
@@ -335,7 +335,7 @@ function combine(frames::Vararg{<:AbstractArray}; method = median)
     return reshape(method(stack(frames), dims = dim), shape)
 end
 
-combine(frames::AbstractVector{<:AbstractArray}; method = median) = combine(frames..., method = method)
+combine(frames::AbstractVector; method = median) = combine(frames..., method = method)
```

Comment:
I think this might be slightly overspecified? It should work fine with just 
```suggestion
combine(frames; method = median) = combine(frames..., method = method)
```

Reply:
Not specifying `AbstractVector` makes this one go wrong:
```
x = [reshape(1:4, (2,2)) for i = 1:10]

@which combine(x)
```
The `combine` goes to:
`combine(frames::AbstractArray...; method) in Main at /home/siddharth/.julia/dev/CCDReduction/src/methods.jl:332` whereas it should have gone to line number 338.

Reply:
I see. The problem I have with forcing the arg to be an `AbstractVector` is shown by the following:
```julia
x = (reshape(1:4, 2, 2) for i in 1:10)
combine(x) # MethodError
```

Let's leave how it was for now and revisit this later. I'm not too concerned about it right now. This is because the point for using a generator would be to avoid having everything in memory at the same time, but we have to do that anyways to combine them (except for certain cases like using `sum` or `mean` for the combination method).

Reply:
Actually, I think we can solve this-
all we (might) have to do is change [this signature](https://github.com/JuliaAstro/CCDReduction.jl/blob/c75d5819e1e04d5d9793ec0c44295c14df7a3cf2/src/methods.jl#L331)
to be

```julia
combine(frame::Vararg{<:AbstractArray{<:Number}}, ...)
```
This way, the `[reshape(1:4, (2,2)) for i = 1:10]` example won't dispatch here, because even though it is an `AbstractArray` the element type is not a number.

Reply:
Yes, this worked! :tada: